File: mot-web-frontend/module/DvsaClient/src/DvsaClient/Entity/Address.php
<?php

namespace DvsaClient\Entity;

/**
 * Class Address
 *
 * @package DvsaClient\Entity
 */
class Address
{

    private $addressLine1;
    private $addressLine2;
    private $addressLine3;
    private $postcode;
    private $county;
    private $town;
    private $country;

    /**
     * @param string $addressLine1
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setAddressLine1($addressLine1)
    {
        $this->addressLine1 = $addressLine1;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getAddressLine1()
    {
        return $this->addressLine1;
    }

    /**
     * @param string $addressLine2
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setAddressLine2($addressLine2)
    {
        $this->addressLine2 = $addressLine2;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getAddressLine2()
    {
        return $this->addressLine2;
    }

    /**
     * @param string $addressLine3
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setAddressLine3($addressLine3)
    {
        $this->addressLine3 = $addressLine3;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getAddressLine3()
    {
        return $this->addressLine3;
    }

    /**
     * @param string $postcode
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setPostcode($postcode)
    {
        $this->postcode = $postcode;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getPostcode()
    {
        return $this->postcode;
    }

    /**
     * @param string $town
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setTown($town)
    {
        $this->town = $town;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getTown()
    {
        return $this->town;
    }

    /**
     * @param string $country
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setCountry($country)
    {
        $this->country = $country;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getCountry()
    {
        return $this->country;
    }

    /**
     * @param string $county
     *
     * @return $this
     * @codeCoverageIgnore
     */
    public function setCounty($county)
    {
        $this->county = $county;
        return $this;
    }

    /**
     * @return string
     * @codeCoverageIgnore
     */
    public function getCounty()
    {
        return $this->county;
    }
}

File: mot-selenium/src/main/java/uk/gov/dvsa/domain/model/Address.java
package uk.gov.dvsa.domain.model;

import uk.gov.dvsa.helper.ContactDetailsHelper;

public class Address {
    private String line1;
    private String line2;
    private String line3;
    private String town;
    private String county;
    private String postcode;


    public Address() {
        line1 = ContactDetailsHelper.getAddressLine1();
        line2 = ContactDetailsHelper.getAddressLine2();
        line3 = ContactDetailsHelper.getAddressLine3();

        town = ContactDetailsHelper.getCity();
        county = ContactDetailsHelper.getCity();
        postcode = ContactDetailsHelper.getPostCode();
    }

    public String getLine1() {
        return line1 != null ? line1 : "";
    }

    public String getLine2() {
        return line2 != null ? line2 : "";
    }

    public String getLine3() {
        return line3 != null ? line3 : "";
    }

    public String getTown() {
        return town != null ? town : "";
    }

    public String getCounty() {
        return county != null ? county : "";
    }

    public String getPostcode() {
        return postcode != null ? postcode : "";
    }

    public String getShortAddress() {
        return getLine1() + ", " + getTown();
    }

    public String getAddress() {
        return getLine1() + ", " + getLine2() + ", " + getTown() + ", " + getPostcode();
    }
}
